src/Http/Resources/SchemaResource.php

```
@@ -0,0 +1,85 @@
+	}
+
+	/**
+	 * Transform the resource into an array.
+	 *
+	 * @since 1.1.0
+	 *
+	 * @param  Request  $request  The incoming request.
+	 *
+	 * @return array<string, mixed>
+	 */
+	public function toArray( Request $request ): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)